src/style.css

```
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 25px;
+  height: 25px;
```

```suggestion
  width: 16px;
  height: 16px;
```
and add the margin between stars to fix the tests

---

feel free to ask for some help in the chat